doc/topics/jinja/index.rst

```diff
@@ -1542,13 +1542,13 @@ Example:
 
 .. code-block:: jinja
 
-regex_escape = {{ 'https://example.com?foo=bar%20baz' | regex_escape }}
+  regex_escape = {{ 'https://example.com?foo=bar%20baz' | regex_escape }}
```

Comment:
Ah, these should be 4 space instead of 2 here. :)

Reply:
Ok, what is the right procedure tu update this PR (or I need to create a new one?)?
sorry to bother you with these question but I'm a newbie of git/github/PR/etc.

And I found in the same file a lot of code-block jinja with only 2 space:
```
Return the union of two lists.

Example:

.. code-block:: jinja

  {{ [1, 2, 3] | union([2, 3, 4]) | join(', ') }}

Returns:
```

Do you want me to try to fix all the file?

And last question: there is some way to test the modification I do with the documentation? some kind of preview?

Thank you very much for your patience

Reply:
@mvivaldi Oh! You know what, it looks like the docs are rendering just fine with 2 spaces instead of 4. My mistake. I thought RST syntax needed 4 spaces, but I was wrong so this pull request is just fine as-is.

> what is the right procedure to update this PR

When you create a pull request from the command line and add new commits to it, you just have to push the commits to your remote branch. The pull request will update automatically. There's some more information about that [here](https://help.github.com/articles/about-pull-requests/), if you like. 

If you created the changes from the GitHub website directly, you need to make sure you commit the change to the same branch (`filters-doc` in this case). See the last paragraph of [this article](https://github.com/WGBH/pbucore/wiki/Contributing-to-the-project-through-Github-web-interface) for more information.

Hopefully that information helps! I am happy to answer any other questions.

> Do you want me to try to fix all the file?

No, this file is fine the way it is.

> there is some way to test the modification I do with the documentation? some kind of preview?

You can build the docs locally and preview them by following [these instructions](https://docs.saltstack.com/en/latest/topics/development/conventions/documentation.html#building-the-documentation). The initial build takes a while, but once that is finished, subsequent builds are faster.

Reply:
Thank you for all the informations!